sc-memory/sc-memory/sc_keynodes.hpp

```diff
@@ -238,6 +238,9 @@ class _SC_EXTERN ScKeynodes : public ScObject
       "action_finished_with_error",
       ScType::NodeConstClass};
   _SC_EXTERN static inline ScKeynode const nrel_result{"nrel_result", ScType::NodeConstNoRole};
+  _SC_EXTERN static inline ScKeynode const nrel_expected_execution_time_in_milliseconds{
+      "nrel_expected_execution_time_in_milliseconds",
```

Comment:
Expected execution time does not reflect the meaning of this parameter. This param about waiting

Reply:
I took this relation from [here](https://libeldoc.bsuir.by/handle/123456789/48018) page 44
when someone invokes this method he expects for action to be finished in that time so he passes this time and waits
I can add 'max' to the name so it'll be `nrel_max_expected_execution_time_in_milliseconds`